packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/SelectedEntriesModal/index.js

```diff
@@ -428,11 +431,11 @@ const SelectedEntriesModal = ({ onToggle }) => {
           try {
             schema.validateSync(entry, { abortEarly: false });
 
-            return { entity: entry };
+            return entry;
           } catch (e) {
             return {
-              entity: entry,
-              errors: getYupInnerErrors(e),
+              ...entry,
+              validationErrors: getYupInnerErrors(e),
```

Comment:
How come we're changing this? how do we know someone doesn't have `validationErrors` in the content-type as an attribute? 

We spoke previously about this, hence the release it was, the way it was....

Reply:
The problem is that I need a plain object returning from the API and not an object with the structure
{
  entity: { ... },
  errors: ...
}
because otherwise I had a problem [here](https://github.com/strapi/strapi/blob/2aee20a2172832c866e397b68364bfc5fad2c535/packages/core/helper-plugin/src/components/Table/index.js#L156) in the select deselect all handler in the Table 
But I understand your point, I need to figure it out if there is another possible solution without touching the Table component

Reply:
The issue is the Table from the helper-plugin is assuming the row object _is_ the entity and not {entity: {}, errors: {}}, so it was looking for row.id in places instead of row.entity.id. 

This use of the Table is the exception to the rule so I don't think we should make the change there. 

To address your concern what do you think about removing the validation errors at the entity level and mapping the errors to a new object, something like:

```
results: {...entities}
validationErrors: {[entityId]: {...errors}}
```

Reply:
Yeah i think that might be the only way to do it! 

Maybe the `Table` component is too tied to the concept of an entity, what do you think @markkaylor? Maybe we write a todo to refactor and make it "looser"?

Reply:
Yup I agree it would be best if it was more flexible. I went ahead and created a ticket in our backlog where we can start collecting some ideas to improve the table composable table 👍